docs/sources/reference/commandline/cli.md

```diff
@@ -533,6 +533,10 @@ considered as normal files. Filepaths in .dockerignore are absolute with
 the current directory as the root. Wildcards are allowed but the search
 is not recursive.
 
+When the `URL` references a Dockerfile, or a Dockerfile is read from 
+`STDIN`, then the `-f` option will have no effect and the contents of the
+Dockerfile will be placed in a file called `Dockerfile`.
```

Comment:
@duglin The problem is there are two many points being made in sentence that does not have the structure to present them all, thus my suggestion to break it into. This would allow to sufficiently explain what you mean by "...the contents of the Dockerfile will be placed in a file called `Dockerfile`", a tautology that will baffle the reader. Other than a typo'd lower-case Dockerfile, what are the issues you see with my suggested revision?